tests/docker-images/latest-version-image/Dockerfile

```diff
@@ -27,12 +27,14 @@ RUN cd /go/src/github.com/apache/pulsar/pulsar-function-go/pf && go install
 RUN cd /go/src/github.com/apache/pulsar/pulsar-function-go/examples && go install ./...
 
 # Reference pulsar-all to copy connectors from there
-FROM apachepulsar/pulsar-all:latest as pulsar-all
+ARG PULSAR_ALL_IMAGE
+FROM $PULSAR_ALL_IMAGE as pulsar-all
 
 ########################################
 ###### Main image build
 ########################################
-FROM apachepulsar/pulsar:latest
+ARG PULSAR_IMAGE
+FROM $PULSAR_IMAGE
```

Comment:
It seems that `PULSAR_ALL_IMAGE` and `PULSAR_IMAGE` aren't passed as args for building the docker image